lvsfunc/aa.py

```diff
@@ -232,18 +236,21 @@ def upscaled_sraa(clip: vs.VideoNode,
             width = get_w(height, aspect_ratio=clip.width / clip.height)
         else:
             width = clip.width
-
+            
+    nnedi3 = core.nnedi3cl.NNEDI3CL if nnedi3_cl else core.nnedi3.nnedi3
+    eedi3 = core.eedi3m.EEDI3CL if eedi3_cl else core.eedi3m.EEDI3
+    
     # Nnedi3 upscale from source height to source height * rounding (Default 1.5)
-    up_y = core.nnedi3.nnedi3(luma, 0, 1, 0, **nnargs)
+    up_y = nnedi3(luma, 0, 1, 0, **nnargs)
     up_y = core.resize.Spline36(up_y, height=ssh, src_top=.5)
     up_y = core.std.Transpose(up_y)
-    up_y = core.nnedi3.nnedi3(up_y, 0, 1, 0, **nnargs)
+    up_y = nnedi3(up_y, 0, 1, 0, **nnargs)
     up_y = core.resize.Spline36(up_y, height=ssw, src_top=.5)
 
     # Single-rate AA
-    aa_y = core.eedi3m.EEDI3(up_y, 0, 0, 0, sclip=core.nnedi3.nnedi3(up_y, 0, 0, 0, **nnargs), **eeargs)
+    aa_y = eedi3(up_y, 0, 0, 0, sclip=core.nnedi3.nnedi3(up_y, 0, 0, 0, **nnargs), **eeargs)
```

Comment:
Shouldn't this line use `nnedi3` as well? (same below)

Reply:
Yes, you are right. My fault.